frontends/mit-open/src/page-components/ManageListDialogs/ManageListDialogs.tsx

```diff
@@ -379,11 +343,12 @@ const DeleteUserListDialog = NiceModal.create(
       })
       hideModal()
     }, [destroyList, hideModal, userList])
+
     return (
       <Dialog
         {...NiceModal.muiDialogV5(modal)}
         onConfirm={handleConfirm}
-        title="Delete User List"
+        title="Delete List"
```

Comment:
When testing out the "delete list" functionality after a rebase, the list is successfully deleted but I end up at a 404 afterward. This is because the routing surrounding the dashboard has changed. "My Lists" is now at `/dashboard/my-lists`, and when you click one you are sent to `/dashboard/my-lists/23` for example. So, when you delete the list, a route to it no longer exists and you get a 404. After deleting the list, we should redirect back to `/dashboard/my-lists`.

Reply:
I'll get that one fixed, thanks @gumaerc.

We've actually lost the delete functionality for the moment as the designs for your recent PR removed the icon menu from the cards, while this one adds a delete button into the edit dialog.

Reply:
I've added the fix to navigate back to the list page in https://github.com/mitodl/mit-open/pull/1356/commits/f42a3bcf5b059a34a9eb52c0509e2847aacd6a49.